salt/environment_settings.yml

```diff
@@ -326,6 +326,10 @@ environments:
         num_instances: 3
         business_unit: operations
         healthcheck: /health
+      apps-es:
```

Comment:
Nitpick: Should it be called `app-es` since it's already under the `apps` umbrella?

Reply:
I think the `apps` designation is accurate since it is intended to be multi-tenant